homework-submission/react-homework-w01/src/dynamiclist.json

```diff
@@ -0,0 +1,20 @@
+[

```

Comment:
If you didn't end up using the json file, maybe remove it from your code. Can be confusing for people reading the code.

Reply:
Its gone :) I actually included it in hopes of fixing the commented out code later :/